python3/forwardfinancing.py

```diff
@@ -0,0 +1,115 @@
+import requests
+import os
+import json
+import base64
+
+# Since you don't want to hard-code an api key, read it from environment var
```

Comment:
`#Get api key from environment variable` (less judgment there?)

Reply:
I don't think we can assume that they have their api key saved as an env var. Do you think there is a better way to phrase that?

Reply:
It's in the `Requirements` so I will trust them to have read them before they start.
`#Read the api key previously exported to your environment as part of the set-up`? This or equivalent.

Reply:
Oh yeah, i forgot I wrote that 😅

Reply:
I forget everything I type as soon as I type it so 👍